src/goto-symex/goto_symex_state.cpp

```diff
@@ -288,8 +290,9 @@ static void assert_l1_renaming(const exprt &expr)
   #if 1
   if(check_renaming_l1(expr))
   {
-    std::cerr << expr.pretty() << '\n';
-    UNREACHABLE;
+    std::stringstream ss;
+    ss << "l1 renaming failed: \n" << expr.pretty();
+    DATA_INVARIANT(false, ss.str());
```

Comment:
For simple things like this, std::string op+ is enough, no real need for sstream